### DISCUSSION TITLE
How to run Daisy UI Locally to test a fix?

### DISCUSSION THREAD

**Discussion by kenghtt** [saadeghi/daisyui#1768] [General]
Hello!

I'm new to contributing to opensource. I found an interesting small issue with scrolling when selecting the theme dropdown, 

I think it's a small fix. I have forked this project, but cannot seem to find a way to run it locally so that I can test the fix? 

I tried running 'npm run dev' but spits out this into the console 'The system cannot find the file specified.'

Any help is appreciated. Thanks.

**Comment by saadeghi** [comment]
This guide should help:  
https://github.com/saadeghi/daisyui/blob/master/.github/CONTRIBUTING.md#to-run-the-documentation-site-on-local  

This repo contains the source of `daisyui` NPM package on `/src` and the document site on `/src/docs` running the site on local uses the local build of the package (instead of the published version) so you should:
1. install the package dependencies: `npm install`
2. build the package locally `npm run build`
3. install document site dependencies:  `npm run install:docs`
4. then you can run the site: `npm run dev`


Let me know if you have any questions.
